### PR TITLE
Update de.ini

### PR DESCRIPTION
**Pull request by ReinforceZwei**
@ORelio Do we merge it directly into the repo or ask for translator to help translating before merging?

### CODE REVIEW

File: MinecraftClient/Resources/lang/de.ini [hunk 1]
```diff
@@ -39,6 +39,8 @@ mcc.session_fail=Konnte Sitzung nicht überprüfen.
 mcc.server_protocol=§8Server-Version : {0} (protocol v{1})
 mcc.with_forge=, mit Forge
 mcc.handshake=§8Handshake erfolgreich. (Server ID: {0})
+mcc.realms_available=You have access to the following Realms worlds
```
Comment (Daenges) [inline MinecraftClient/Resources/lang/de.ini:42]:
```suggestion
mcc.realms_available=Du hast Zugang zu den folgenden Realms Welten
```

File: MinecraftClient/Resources/lang/de.ini [hunk 2]
```diff
@@ -39,6 +39,8 @@ mcc.session_fail=Konnte Sitzung nicht überprüfen.
...
+mcc.realms_join=Use realms:index as server IP to join the Realms world
```
Comment (Daenges) [inline MinecraftClient/Resources/lang/de.ini:43]:
```suggestion
mcc.realms_join=Nutze realms:index als Server IP um der Realms Welt beizutreten
```

File: MinecraftClient/Resources/lang/de.ini [hunk 3]
```diff
@@ -82,6 +84,10 @@ error.forge_encrypt=§8Forge StartEncryption Handshake konnte nicht erfolgreich
 error.setting.str2int=Konnte '{0}' nicht in einen Int umwandeln. Bitte überprüfe die Einstellungen.
 error.http_code=§8Es gab einen Fehlercode vom Server: {0}
 error.auth=§8Es gab einen Fehlercode vom Server, während die Authentication erneuert wurde: {0}
+error.realms.ip_error=Cannot retrieve the server IP of your Realms world
+error.realms.access_denied=This Realms world does not exist or access was denied
```
Comment (Daenges) [inline MinecraftClient/Resources/lang/de.ini:88]:
```suggestion
error.realms.access_denied=Diese Realms Welt exsistiert nicht oder der Zugang wurde verweigert
```

File: MinecraftClient/Resources/lang/de.ini [hunk 4]
```diff
@@ -82,6 +84,10 @@ error.forge_encrypt=§8Forge StartEncryption Handshake konnte nicht erfolgreich
...
+error.realms.server_unavailable=Realms server may require some time to start up. Please retry again later.
```
Comment (Daenges) [inline MinecraftClient/Resources/lang/de.ini:89]:
```suggestion
error.realms.server_unavailable=Der Realms Server braucht ein wenig Zeit zum starten. Bitte versuche es später erneut.
```

File: MinecraftClient/Resources/lang/de.ini [hunk 5]
```diff
@@ -82,6 +84,10 @@ error.forge_encrypt=§8Forge StartEncryption Handshake konnte nicht erfolgreich
...
+error.realms.disabled=Trying to join a Realms world but Realms support is disabled in config
```
Comment (Daenges) [inline MinecraftClient/Resources/lang/de.ini:90]:
```suggestion
error.realms.disabled=Versuche der Realms Welt beizutreten, allerdings wurde der Realms Support in der Config deaktiviert
```

**Review by Daenges — COMMENTED** [review]
I tried to translate everything.

**Comment by ORelio** [comment]
Thanks @ReinforceZwei and @Daenges! Maybe @cyb3rko could have a look as in #1526?

File: MinecraftClient/Resources/lang/de.ini [hunk 6]
```diff
@@ -39,6 +39,8 @@ mcc.session_fail=Konnte Sitzung nicht überprüfen.
 mcc.server_protocol=§8Server-Version : {0} (protocol v{1})
 mcc.with_forge=, mit Forge
 mcc.handshake=§8Handshake erfolgreich. (Server ID: {0})
+mcc.realms_available=Du hast Zugang zu den folgenden Realms Welten
```
Comment (cyb3rko) [inline MinecraftClient/Resources/lang/de.ini:42]:
```suggestion
mcc.realms_available=Du hast Zugang zu den folgenden Realms-Welten
```

Comment (cyb3rko) [reply MinecraftClient/Resources/lang/de.ini:42]: https://www.minecraft.net/de-de/realms/faq/ in the last paragraph

File: MinecraftClient/Resources/lang/de.ini [hunk 7]
```diff
@@ -39,6 +39,8 @@ mcc.session_fail=Konnte Sitzung nicht überprüfen.
...
+mcc.realms_join=Nutze realms:index als Server IP um der Realms Welt beizutreten
```
Comment (cyb3rko) [inline MinecraftClient/Resources/lang/de.ini:43]:
```suggestion
mcc.realms_join=Nutze realms:index als Server-IP, um der Realms-Welt beizutreten
```

File: MinecraftClient/Resources/lang/de.ini [hunk 8]
```diff
@@ -82,6 +84,10 @@ error.forge_encrypt=§8Forge StartEncryption Handshake konnte nicht erfolgreich
 error.setting.str2int=Konnte '{0}' nicht in einen Int umwandeln. Bitte überprüfe die Einstellungen.
 error.http_code=§8Es gab einen Fehlercode vom Server: {0}
 error.auth=§8Es gab einen Fehlercode vom Server, während die Authentication erneuert wurde: {0}
+error.realms.ip_error=Konnte die Server IP deiner Realms Welt nicht abfragen
```
Comment (cyb3rko) [inline MinecraftClient/Resources/lang/de.ini:87]:
```suggestion
error.realms.ip_error=Konnte die Server-IP deiner Realms-Welt nicht abfragen
```

File: MinecraftClient/Resources/lang/de.ini [hunk 9]
```diff
@@ -82,6 +84,10 @@ error.forge_encrypt=§8Forge StartEncryption Handshake konnte nicht erfolgreich
...
+error.realms.access_denied=Diese Realms Welt exsistiert nicht oder der Zugang wurde verweigert
```
Comment (cyb3rko) [inline MinecraftClient/Resources/lang/de.ini:88]:
```suggestion
error.realms.access_denied=Diese Realms-Welt existiert nicht oder der Zugang wurde verweigert
```

File: MinecraftClient/Resources/lang/de.ini [hunk 10]
```diff
@@ -82,6 +84,10 @@ error.forge_encrypt=§8Forge StartEncryption Handshake konnte nicht erfolgreich
...
+error.realms.server_unavailable=Der Realms Server braucht ein wenig Zeit zum starten. Bitte versuche es später erneut.
```
Comment (cyb3rko) [inline MinecraftClient/Resources/lang/de.ini:89]:
```suggestion
error.realms.server_unavailable=Der Realms-Server braucht ein wenig Zeit zum starten. Bitte versuche es später erneut.
```

File: MinecraftClient/Resources/lang/de.ini [hunk 11]
```diff
@@ -82,6 +84,10 @@ error.forge_encrypt=§8Forge StartEncryption Handshake konnte nicht erfolgreich
...
+error.realms.disabled=Versuche der Realms Welt beizutreten, allerdings wurde der Realms Support in der Config deaktiviert
```
Comment (cyb3rko) [inline MinecraftClient/Resources/lang/de.ini:90]:
```suggestion
error.realms.disabled=Versuche der Realms-Welt beizutreten, allerdings wurde der Realms-Support in der Config deaktiviert
```

**Review by cyb3rko — CHANGES_REQUESTED** [review]
Fixes some smaller issues

**Comment by ORelio** [comment]
Thanks @ReinforceZwei, @Daenges and @cyb3rko! Is it ready to merge?

**Comment by ReinforceZwei** [comment]
@cyb3rko reacted with thumbs up. I guess it is ready for merging.

**Pull request merged**